src/polyfills/event.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright 2017 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ *@constructor
+  */
```

Comment:
Actually, this isn't what made it pass the type-check. It was not assigning directly to win.Event and instead using Object.defineProperty.

I included this because I read in the docs that `@constructor` should be used if a function will be a constructor. I don't really understand the compiler well enough to know if this makes a difference though.

I will correct the spacing, but I could also completely remove this if it's not helpful.